.circleci/config.yml

```diff
@@ -28,137 +22,94 @@ commands:
       - run: |
           GOOS=<< parameters.os >> \
           GOARCH=<< parameters.arch >> \
-          buildevents cmd $CIRCLE_WORKFLOW_ID $BUILDEVENTS_SPAN_ID go_build -- \
           go build -ldflags "-X main.BuildID=${CIRCLE_TAG}" \
           -o $GOPATH/bin/honeytail-<< parameters.os >>-<< parameters.arch >> \
           .
 
 jobs:
-  setup:
-    executor: linuxgo
-    steps:
-      - buildevents/start_trace
-
-  watch:
-    executor: linuxgo
-    steps:
-      - buildevents/watch_build_and_finish
-
-  test:
-    executor: linuxgo
+  build:
+    docker:
+      - image: circleci/golang:1.13
     steps:
-      - buildevents/with_job_span:
-          steps:
-            - checkout
-            - buildevents/berun:
-                bename: go_test
-                becommand: go test --timeout 10s -v ./...
+      - checkout
+      - run: go test --timeout 10s -v ./...
```

Comment:
If I'm reading this correctly, the `build` job is now responsible for running the tests and there is no `test` job any more, right? I'd expect this job to be named `test`. 😄

Reply:
You are making sense.